components/organisms/SearchDialog/search-dialog.tsx

```diff
@@ -70,13 +70,10 @@ const SearchDialog = () => {
   }, [debouncedSearchTerm]);
 
   return (
-    <div className="fixed left-0 top-0 z-[55] p-5 w-full h-full flex justify-center bg-white/30">
+    <div className="fixed left-0 top-0 z-auto p-5 w-full h-full flex justify-center bg-white/30">
```

Comment:
Wondering if this won't cause a breaks somewhere in the app 🤔

Reply:
I will do a video on what the UI looks like now.